packages/jest-cli/src/runJest.js

```diff
@@ -29,14 +29,6 @@ import FailedTestsCache from './FailedTestsCache';
 import {JestHook} from 'jest-watcher';
 import collectNodeHandles from './collectHandles';
 
-const setConfig = (contexts, newConfig) =>
-  contexts.forEach(
-    context =>
-      (context.config = Object.freeze(
```

Comment:
It's already frozen in [`jest-config`](https://github.com/facebook/jest/blob/a1f3e2cda489e262bd0a258a51c99f6a76812e1e/packages/jest-config/src/index.js#L99). We only freeze it here because we're creating a new instance, which isn't necessary anymore.